source/shared/cpp/ObjectModel/BaseCardElement.cpp

```diff
@@ -228,4 +238,6 @@ void BaseCardElement::DeserializeBaseProperties(ParseContext& context, const Jso
     element->SetSeparator(ParseUtil::GetBool(json, AdaptiveCardSchemaKey::Separator, false));
     element->SetSpacing(ParseUtil::GetEnumValue<Spacing>(json, AdaptiveCardSchemaKey::Spacing, Spacing::Default, SpacingFromString));
     element->SetAreaGridName(ParseUtil::GetString(json, AdaptiveCardSchemaKey::AreaGridName, "", false));
```

Comment:
We are already setting default value to empty string here. So we will always get a value for area grid.  Do we need another method?

Reply:
optional string is not supported in java. swig convert it to this type SWIGTYPE_p_std__optionalT_std__string_t that we can't use.

Reply:
Changed it to non optional String with default value.

Reply:
Reverted "non optional String with default value."  as it was breaking iOS build. Will take it as a follow up item.